app/views/api/v1/accounts/portals/_portal.json.jbuilder

```diff
@@ -17,3 +17,9 @@ json.portal_members do
     end
   end
 end
+
+json.meta do
+  json.articles_count portal.articles.published.size
```

Comment:
Shouldn't we return all the articles?

Reply:
![Uploading Screenshot 2022-07-27 at 2.58.46 PM.png…]()

Reply:
![Screenshot 2022-07-27 at 3 00 01 PM](https://user-images.githubusercontent.com/12408980/181213571-0cb44ece-8169-4b21-a1a2-7d02da1c6196.png)

Reply:
It was mentioned in one of the tickets about the published articles count.